text-generation-inference/server/text_generation_server/jetstream_pt_support/generator.py

```diff
@@ -330,6 +331,9 @@ def warmup(self, batch: Batch) -> int:
         # Counter-intuitively, now we ignore the input batch. Instead, we create dummy batches to cover all possible
         # batch sizes and sequence lengths.
         seq_len = self.model.config.sequence_length
+        if os.environ.get("SKIP_WARMUP", "0") == "1":
+            logger.debug("Skipping warmup")
```

Comment:
Does this is used mostly for debug only? Or can it be turned on for other reasons? In the later case I would use `logger.warning` if not, `logger.debug` is fine

Reply:
I would say it's only for debugging. Warmup will check if model can fit in memory and prepare inference so prefill and decode is rather fast afterwards, but it can last around 4-5 minutes, so it can be annoying for debugging the container.